Add Wide (Golang IDE)

Gys opened the pull request:
Fixes #32

styfle commented on README.md:
```diff
@@ -20,6 +20,7 @@ A curated list of awesome online implementations of integrated development envir
 * [Snack Expo](https://snack.expo.io) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/expo/snack-sdk) - Run your React Native javascript in the browser and on your mobile iOS or Android
 * [Codeanywhere](https://codeanywhere.com/) - Run many languages in a container with pre-built environment and a terminal
 * [WebAssembly Studio](https://webassembly.studio/) [<img title="Open Source" width="16" src="https://cdn.jsdelivr.net/npm/simple-icons@1.2.7/icons/github.svg" />](https://github.com/wasdk/WebAssemblyStudio) - Run C, Rust, Wat, or AssemblyScript code as WebAssembly in the browser
+* [Wide](https://github.com/b3log/wide) - Run Go in the browser (including mobile) with expression-info, autocomplete and jump to declaration - self hosted or use cloud (requires registration)
```
Can you remove the "self hosted or use cloud" part and instead use the open source icon to link to the github repo (like the others above)?

Gys commented:
I changed the Wide description. I also removed the registration required part, because others, for example c9 (AWS even needs payment info) and CodeAnywhere, need registration as well. And that is not mentioned either.

styfle commented:
Great, thanks 👍

styfle commented:
I added a "Sign Up Required" icon in PR #42 so that this is a little more clear to the reader.
